bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
@aantron No, pinning doesn't imply dev (why should it ? You can pin to a source package containing other versions).